applications/DEMApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -104,6 +104,8 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
         .def(py::init< ExplicitSolverSettings&, double, int, double, int, ParticleCreatorDestructor::Pointer,DEM_FEM_Search::Pointer, SpatialSearch::Pointer, Parameters>())
         .def("Solve", &ExplicitSolverStrategy::Solve)
         .def("Initialize", &ExplicitSolverStrategy::Initialize)
+        .def("InitializeSolutionStep", &ExplicitSolverStrategy::InitializeSolutionStep)
+        .def("FinalizeSolutionStep", &ExplicitSolverStrategy::FinalizeSolutionStep)
```

Comment:
if `ExplicitSolverStrategy` derives from `SolvingStrategy` then this should already be exposed there I think

Reply:
It does not derive from it atm, but maybe we could consider doing so.

Reply:
It used to derive from SolvingStrategy, but after several years without any need of using the template alguments (LinearSolver, Space, ...) keeping the template was quite annoying when subderiving our strategy. So we decided to separate from the template.